bokehjs/src/coffee/models/transforms/jitter.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+Transform = require "./transform"
+p = require "../../core/properties"
+
+class Jitter extends Transform.Model
+  initialize: (attrs, options) ->
+    super(attrs, options)
+
+  props: ->
+    return _.extend {}, super(), {
+      mean:         [ p.Number, 0        ]
+      width:        [ p.Number, 1        ]
+      distribution: [ p.String, 'uniform']
```

Comment:
is also possible to define enums (which will validate) on JS side:

https://github.com/bokeh/bokeh/blob/master/bokehjs/src/coffee/core/enums.coffee
https://github.com/bokeh/bokeh/blob/master/bokehjs/src/coffee/core/properties.coffee#L153